### ISSUE TITLE
Get iSCSI multipathing working without disabling local drive

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/esi#257]
I was getting bunch of I/O errors when I turned on iSCSI multipathing on a test node. I followed a gut feeling and disabled the local drive of the system and then it worked. I know there's a way to blacklist certain block devices from being discovered and added to multipathing so we should investigate that so we don't have to disable the local hard drive of a server.

**Comment by naved001** [comment]
I did get a system to boot into multipathed drive without having to disable the local drive. 

All I did was made sure that the only wwid listed in /etc/multipath/wwid was of the iSCSI drives (and the local drive was not listed there). 

```
[root@host-10-21-0-152 ~]# multipath -a /dev/sdb
wwid '3600140575ddc703a953425ebe5242e45' added
```

and that was that. 

I want to repeat this again to make sure something else wasn't at play.

**Comment by naved001** [comment]
okay, I tested this a couple of time, and all you gotta do is make sure that `/etc/multipath/wwid` and `/etc/multipath/bindings` don't have WWIDs listed of other drives besides the one you want to multipath. I just deleted any entries from those files. Ideally, `find_multipaths` option in `/etc/multipath.conf` should automatically figure out which drives can be multipathed, but didn't seem to work. I manually added my boot drive by running `multipath -a /dev/sdX`,

After that, when creating the initramfs include `/etc/multipath` with `dracut --force -H --add multipath --include /etc/multipath`. 

This survived reboots i.e. after a reboot my boot drive was still multipathed. However, when I take this volume and boot a different node with it, multipathing wasn't enabled I had to redo the steps to enable it.
